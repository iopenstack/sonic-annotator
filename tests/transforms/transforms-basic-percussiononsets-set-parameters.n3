@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
@prefix vamp: <http://purl.org/ontology/vamp/>.
@prefix examples: <http://vamp-plugins.org/rdf/plugins/vamp-example-plugins#>.
@prefix : <#>.

:transform0 a vamp:Transform;
	vamp:plugin examples:percussiononsets ;
	vamp:output examples:percussiononsets_output_onsets ;
	vamp:step_size "512";
 	vamp:block_size "1024";
	vamp:parameter_binding :param0 ;
	vamp:parameter_binding :param1 .

:param0 a vamp:ParameterBinding;
	vamp:parameter [ vamp:identifier "sensitivity" ];
	vamp:value "60".

:param1 a vamp:ParameterBinding;
	vamp:parameter examples:percussiononsets_param_threshold;
	vamp:value "2.6".

 
