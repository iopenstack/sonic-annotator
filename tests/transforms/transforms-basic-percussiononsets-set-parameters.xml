<transform
    id="vamp:vamp-example-plugins:percussiononsets:onsets"
    stepSize="512"
    blockSize="1024">
  <parameter name="sensitivity" value="60"/>
  <parameter name="threshold" value="2.6"/>
</transform>
