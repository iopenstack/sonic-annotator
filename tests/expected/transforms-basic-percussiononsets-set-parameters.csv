"./audio/3clicks8.wav",0.011609977
,0.162539682
,0.290249433
,0.394739229
,0.777868481
,1.648616780
,2.414875283
,3.041814059
,3.134693877
,3.157913832
,3.599092970
,3.831292517
,4.504671202
,4.748480725
